static/css/page-signup.less

```
@@ -153,9 +153,19 @@
     color: @grey;
   }
 
+  .input {
+    position: relative;
+  }
+
   .password-visibility-toggle {
-    float: right;
-    color: @grey;
+    position: absolute;
+    right: 5px;
+    top: 5px;
+  }
+
+  .password-visibility-toggle > img {
+    width: 25px;
+    height: 25px;
```

Small tweak to improve the centering a little bit!
```suggestion
    width: 22px;
    height: 22px;
```